water-features.mss

```diff
@@ -70,10 +70,12 @@
       polygon-fill: @land-color;
     }
     #piers-line {
-      line-width: 1.5;
+      line-width: 0.5;
       line-color: @land-color;
-      [zoom >= 13] { line-width: 3; }
-      [zoom >= 16] { line-width: 7; }
+      line-cap: square;
```

Comment:
All roads have round as cap.

Reply:
Round would look best when two sections join at a non-right angle, but I didn't know how people would feel about round pier ends.

Reply:
Yes, the pier ending. You are right